source/dub/init.d

```diff
@@ -168,6 +168,8 @@ private string getUserName()
 		import core.sys.posix.pwd, core.sys.posix.unistd, core.stdc.string : strlen;
 		import std.algorithm : splitter;
 
+		// Bionic doesn't have pw_gecos on ARM
```

Comment:
I have almost no idea about bionic, but it seems defined to me: https://github.com/android/platform_bionic/blob/5725481f1e4cb2e2042b5469b5863023556cf2dd/libc/bionic/grp_pwd.cpp#L422

Is it only undefined for older versions?